### PR TITLE
[wxwidgets] Adding wxWidgets resource compiler (wxrc) to <triplet>/tools/wxwidgets directory

### PR DESCRIPTION
**Pull request by emersonfxbx**
This pull request adds the resource compiler utility to the &lt;triplet&gt;/tools/wxwidgets directory.

### CODE REVIEW

**Comment by PhoebeHui** [comment]
@emersonfxbx, thanks for the PR! Could you also update the CONTROl version?

From
Version: 3.1.3-1

To
Version: 3.1.3
Port-Version: 2

**Review by PhoebeHui — APPROVED** [review]
Thanks for your updates!

File: ports/wxwidgets/portfile.cmake [hunk 1]
```diff
@@ -63,6 +69,9 @@ file(RENAME ${CURRENT_PACKAGES_DIR}/share/wxwidgets/licence.txt ${CURRENT_PACKAG
 if(EXISTS ${CURRENT_PACKAGES_DIR}/lib/mswu/wx/setup.h)
     file(RENAME ${CURRENT_PACKAGES_DIR}/lib/mswu/wx/setup.h ${CURRENT_PACKAGES_DIR}/include/wx/setup.h)
 endif()
+
+vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/${PORT})
```
Comment (ras0219-msft) [inline ports/wxwidgets/portfile.cmake:73]: `vcpkg_copy_tools()` might be more convenient and would handle non-Windows platforms.

[1] https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/vcpkg_copy_tools.md

Comment (PhoebeHui) [inline ports/wxwidgets/portfile.cmake:73]:
```suggestion
vcpkg_copy_tools(TOOL_NAMES wxrc AUTO_CLEAN)
```

File: ports/wxwidgets/portfile.cmake [hunk 2]
```diff
@@ -37,6 +37,12 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 
+# Copy wxrc.exe to ${CURRENT_PACKAGES_DIR}/tools/${PORT}
+if(EXISTS "${CURRENT_PACKAGES_DIR}/bin/wxrc.exe")
+    file(MAKE_DIRECTORY ${CURRENT_PACKAGES_DIR}/tools/${PORT})
+    file(COPY "${CURRENT_PACKAGES_DIR}/bin/wxrc.exe" DESTINATION "${CURRENT_PACKAGES_DIR}/tools/${PORT}")
+endif()
+
```
Comment (PhoebeHui) [inline ports/wxwidgets/portfile.cmake:45]:
```suggestion
```

**Pull request merged**